components/meter/meter-linear.js

```diff
@@ -0,0 +1,53 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { meterStyles } from './meter-styles.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class MeterLinear extends RtlMixin(LitElement)  {
+	static get styles() {
+		return [ meterStyles ];
+	}
+
+	render() {
+		return html `
+			<svg viewBox="-5 -4.5 ${this.width} 9" width="${this.width}">
+				<path class="d2l-meter-full-bar" d="M0,0 h ${this._actualWidth}" />
+				<path class="d2l-meter-progress-bar" d="M0,0 h ${this._computeProgressBarWidth(this.units, this.outOf, this._actualWidth)}" visibility="${this.units ? 'visable' : 'hidden'}"/>
+			</svg>
+		`;
+	}
+
+	static get properties() {
+		return {
+			units: { type: Number, reflect: true },
+			outOf: { type: Number, reflect: true, attribute: 'out-of' },
```

Comment:
Since this is essentially visually a native `<progress>` element, should we just use [its names for its attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress)? i.e. `value` instead of `units` and `max` instead of `outOf`. The name `units` also confuses me a bit... I think of units as "centimetre", "pixel" not as a value.